airmozilla/main/tests/test_views.py

```diff
@@ -1969,8 +1968,8 @@ def test_calendar_data(self):
         while last.month == first.month:
             last += datetime.timedelta(days=1)
 
-        first_ts = int(time.mktime(first.timetuple()))
-        last_ts = int(time.mktime(last.timetuple()))
+        first_ts = first.strftime('%Y-%m-%d')
+        last_ts = last.strftime('%Y-%m-%d')
```

Comment:
The "_ts" suffix came from the days when it was a "time stamp". We can just drop this now.